web-client/src/presenter/computeds/formattedDocketEntries.ts

```diff
@@ -246,6 +246,9 @@ export const getFormattedDocketEntry = ({
       ? 'Unseal to the public and parties of this case'
       : 'Unseal to the public'
     : 'Seal to the public';
+  formattedResult.toolTipText = !formattedResult.isFileAttached
```

Comment:
Missed this on the last PR - worth adding a test here?

Reply:
Good call, it is